apps/src/templates/manageStudents/ManageStudentsActionsCell.jsx

```diff
@@ -0,0 +1,53 @@
+import React, {Component, PropTypes} from 'react';
+import QuickActionsCell from "../tables/QuickActionsCell";
+import PopUpMenu, {MenuBreak} from "@cdo/apps/lib/ui/PopUpMenu";
+import color from "../../util/color";
+import FontAwesome from '../FontAwesome';
+import Button from '../Button';
+
+const styles = {
+  xIcon: {
+    paddingRight: 5,
+  }
+};
+
+class ManageStudentActionsCell extends Component {
+  static propTypes = {
+    id: PropTypes.number.isRequired,
+    isEditing: PropTypes.bool,
+  };
+
+  render() {
+    return (
+      <div>
+        {
+          !this.props.isEditing &&
+          <QuickActionsCell>
+            <PopUpMenu.Item
+              onClick={() => {}}
+            >
+              {"Edit"}
+            </PopUpMenu.Item>
+            <MenuBreak/>
+            <PopUpMenu.Item
+              onClick={()=>{}}
+              color={color.red}
+            >
+              <FontAwesome icon=" fa-times-circle" style={styles.xIcon}/>
```

Comment:
I think you just want `icon="times-circle"`, and the `FontAwesome` component takes care of adding the fa-